app/install/aws-linux.md

```diff
@@ -16,22 +16,17 @@ This guide walks through downloading, installing, and starting **Kong Gateway**
 
 Start by downloading the following package specifically built for the Amazon Linux AMI:
 
-- [Download]({{ site.links.download }}/kong-rpm/download_file?file_path=amazonlinux/amazonlinux2/kong-{{site.data.kong_latest.version}}.aws.amd64.rpm)
+- [CentOS 8]({{ site.links.download }}/kong-rpm/aws/2/kong-{{site.data.kong_latest.version}}.aws.amd64.rpm)
 
 **Enterprise trial users** should download their package from their welcome email and save their license to `/etc/kong/license.json` after step 1.
 
 ## YUM Repositories
 
-You can also install Kong via YUM; follow the instructions on the "Set Me Up"
-section on the page below.
-
-- [RPM Repository](https://bintray.com/kong/kong-rpm)
-
-**NOTE**: ensure that the `baseurl` field of the generated `.repo` file contains
-amazonlinux/amazonlinux; for instance:
+You can also install Kong via YUM
 
 ```
-baseurl=https://kong.bintray.com/kong-rpm/amazonlinux/amazonlinux
+curl {{ site.links.download }}/kong-rpm/aws/2/config.repo | tee /etc/yum.repos.d/kong.repo
+yum install -y kong
```

Comment:
This is two commands, yeah? Should split this up, that's not super clear from the codeblock. Something like

1. Pull down the yum repo:
codeblock
2. Install the Kong repo (is that correct?):
codeblock